tests/test_pipelines_common.py

```diff
@@ -0,0 +1,12 @@
+from diffusers.utils.testing_utils import require_torch
+
+
+@require_torch
+class PipelineTesterMixin:
+    """
+    This mixin is designed to be used with unittest.TestCase classes.
+    It provides a set of common tests for each PyTorch pipeline, e.g. saving and loading the pipeline,
+    equivalence of dict and tuple outputs, etc.
+    """
+
+    pass
```

Comment:
Just a dummy mixin for now (only `@require_torch` is useful), but the purpose is described in the docstring.